cmake/projects/Boost/hunter.cmake

```diff
@@ -21,6 +21,18 @@ hunter_add_version(
     PACKAGE_NAME
     Boost
     VERSION
+    "1.59"
+    URL
+    "http://sourceforge.net/projects/boost/files/boost/1.59.0/boost_1_59_0.tar.gz/download"
```

Comment:
Official boost archives are too bloated. I'm using this repo to store unpacked version and remove docs and tests directories: https://github.com/hunter-packages/boost This makes it like 3 times smaller. Doing it now for 1.59, also there is 1.60 release.